src/app/views/erp-hgrid/erp-hgrid-view.tsx

```diff
@@ -1,59 +1,19 @@
-import { IgrCellTemplateContext, IgrColumn, IgrGridToolbar, IgrGridToolbarActions, IgrGridToolbarExporter, IgrGridToolbarHiding, IgrGridToolbarPinning, IgrGridToolbarTitle, IgrHierarchicalGrid, IgrRowIsland } from 'igniteui-react-grids';
-import { IgrRating } from 'igniteui-react';
-import { useGetTable1List as eRPProductsUseGetTable1List } from '../../hooks/erpproducts-hooks';
-
-import styles from './erp-hgrid-view.module.css';
+import ErpHGrid from '../../../../projects/erp-hierarchical-grid/src/components/ErpHGrid/ErpHGrid';
 import createClassTransformer from '../../style-utils';
 
-function ERPHGrid () {
-  const classes = createClassTransformer(styles);
-  const uuid = () => crypto.randomUUID();
-  const { eRPProductsTable1 } = eRPProductsUseGetTable1List();
-
-  const columnBodyTemplate = (ctx: { dataContext: IgrCellTemplateContext }) => {
-    return (
-      <>
-        <IgrRating value={ctx.dataContext.cell.value} className={classes("rating")}></IgrRating>
-      </>
-    )
-  }
+import styles from './erp-hgrid-view.module.css';
```

Comment:
not used. Can you also remove the `erp-hgrid-view.module.css` content or delete the file itself because it is not used